src/server/serialize.ts

```diff
@@ -26,5 +31,26 @@ export function serializeDocument(doc: Document, opts?: SerializeDocumentOptions
   return html;
 }
 
+/**
+ * Parses the QRL mapping JSON and returns the transform closure.
+ * @alpha
+ */
+function createQrlMapper(qEntryMap: OutputEntryMap) {
+  const symbolManifest = new Map<string, string>();
+
+  const qrlMapper: QrlMapper = (path, symbolName) => {
+    path = symbolManifest.get(symbolName) || path;
+    path = path.slice(0, path.lastIndexOf('.'));
+    return `./${path}#${symbolName}`;
+  };
+
+  for (const symbolName in qEntryMap.mapping) {
```

Comment:
Wonder if we should throw an error here if the object is not in the correct format.